### ISSUE TITLE
Is still possible to use key board navigation in this version ?

### ISSUE THREAD

**Issue by andrisundae** [react-bootstrap-table/react-bootstrap-table2#216]
On the old version http://allenfang.github.io/react-bootstrap-table/example.html#keyboard-navigation, maybe this useful for work with mouseless.

**Comment by AllenFang** [comment]
@andrisundae yes, I will recover all of the useful features in legacy `react-bootstrap-table` ASAP. keyboard navigation is one of them.  I will mention here again if we implement it, thanks!!

**Comment by andrisundae** [comment]
Ok thanks @AllenFang ...you are the best !!

**Comment by kotaro88** [comment]
Dear @AllenFang,
This is awesome project, but I wonder does react-bootstrap-table-2 implement key board navigation ?

Thanks

**Comment by AllenFang** [comment]
@kotaro88 In July or Aug, we will support this feature, thanks

**Comment by AllenFang** [comment]
BTW, very sorry for lately implementation.

**Comment by xxporkoxx** [comment]
Hey @AllenFang how are you doing? Does the feature still in development? Do you have any expected release date? Thanks so much for the module, it is awesome!

**Comment by vinantinaik** [comment]
Hey @AllenFang   I am not able to locate if there is any update on keyboard navigation.
Is it already available?

**Comment by prakis** [comment]
Maybe he already updated it, 

[issue 929](https://github.com/react-bootstrap-table/react-bootstrap-table2/issues/929)

Note: I haven't tested it.

**Comment by ranasarmadiftikhar** [comment]
Hi @AllenFang  Component is awesome. But still basic functionality of navigation key is missing. Please let me know if there is any progress.
